seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -124,6 +124,10 @@ bnb,biswap,1,2022-11-22,0x9050a1e2f7aa392e39195b5b22382d595e94104b72699655b198c4
 bnb,biswap,1,2022-11-22,0xe2efd633720bd29322da7e39857e98b3c9904a1c55f9b3995441160983e06296,119,0xe9e7cea3dedca5984780bafc599bd69add087d56,43.898605549510836795,0x563ca064e41f3b5d80adeecfe49ab375fd7afbef,2756.0281
 bnb,biswap,1,2022-11-22,0x0b9ba3ee4a0b3f4db2baba70be527f5a50aea77423cd921106ec9f0ac8dcf169,115,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.018248512111984361,0xe9e7cea3dedca5984780bafc599bd69add087d56,4.837104443141354752 
 bnb,biswap,1,2022-11-22,0xaf03946d9b98e5d6e841bb50555208801e48d773a8dfdf1ecc203d943c27c603,350,0x2170ed0880ac9a755fd29b2688956bd959f933f8,6.247796965743473757,0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c,0.43849
+bnb,nomiswap,1,2022-11-22,0x2a23d2358cb76eb7be9b10e79e9ccd54ef267f7abd0410df1aa797aa15291a3f,139,0xd32d01a43c869edcd1117c640fbdcfcfd97d9d65,2233.297473430339233608,0x55d398326f99059ff775485246999027b3197955,318.237033328878863377
```

Comment:
@Hosuke @developerfred it appears none of these tx's are from nomiswap. please see [query](https://dune.com/queries/1690943?seed_t6c1ea=test_schema.git_cd0f07af_dex_trades_seed&model_t6c1ea=test_schema.git_cd0f07af_nomiswap_bnb_trades&blockchain_t6c1ea=bnb&project_t6c1ea=nomiswap&version_t6c1ea=1)

Reply:
the model doesn't match to the seed for any of the four rows, repeated across three different tests

Reply:
I will check here now, thanks to the report.

Reply:
Moreover how do we find out the parameter `test_schema.git_cd0f07af_nomiswap_bnb_trades` for the query?
Currently I can only figure out cd0f07af at [$GIT_SHA macro definition in the dbt-test](https://github.com/duneanalytics/spellbook/actions/runs/3595335598/jobs/6054697367#step:8:5)

Reply:
I see, so the model name `test_schema.git_cd0f07af_nomiswap_bnb_trades` can be found [here](https://github.com/duneanalytics/spellbook/actions/runs/3595335598/jobs/6054697367#step:11:41) and the seed name `test_schema.git_cd0f07af_dex_trades_seed` can be found [here](https://github.com/duneanalytics/spellbook/actions/runs/3595335598/jobs/6054697367#step:10:12). I am going to use this query to find out unmatched seeds in other PRs.

Reply:
@Hosuke @jeff-dude Is it necessary to change the router address? or can I make new queries?

Reply:
looks like nomiswap has a proxy router contract: https://bscscan.com/address/0x0df8c8652a1a634538f4593c34a219c0e4848440

I will work with the txids of this proxy

Reply:
NomiSwap Router: https://bscscan.com/address/0xd654953d746f0b114d1f85332dc43446ac79413d#code

Reply:
done

Reply:
> I see, so the model name `test_schema.git_cd0f07af_nomiswap_bnb_trades` can be found [here](https://github.com/duneanalytics/spellbook/actions/runs/3595335598/jobs/6054697367#step:11:41) and the seed name `test_schema.git_cd0f07af_dex_trades_seed` can be found [here](https://github.com/duneanalytics/spellbook/actions/runs/3595335598/jobs/6054697367#step:10:12). I am going to use this query to find out unmatched seeds in other PRs.

yes -- this is important. please use these gh action generated tables to run queries and test output. they persist for ~24 hours and should be taken advantage of, great developer tool for testing